_config.yml

```diff
@@ -47,6 +47,8 @@ collections:
 # Scripts
 scripts:
   - /assets/js/build/main.js
+  - /assets/js/jquery/3.5.1/jquery.min.js
```

Comment:
As mentioned in the other branch, I can't see that we're using jQuery. I think we can remove it.